src/index.html

```diff
@@ -7,8 +7,75 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" id="homePage">
+      <a href="#homePage" class="header__link">
+        <img
+         src="images/logo.png"
+         alt="logo website"
+         class="header__logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+             href="#apple"
+             class="is-active header__links"
+             >Apple</a>
+          </li>
+          <li class="header__item">
+            <a
+             href="#samsung"
+             class="header__links"
+             >Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="header__item">
            <a
             href="#apple"
             class="is-active header__links"
             >Apple</a>
          </li>
          
          <li class="header__item">
            <a
             href="#samsung"
             class="header__links"
             >Samsung</a>
          </li>
```
add the spaces between blocks